types/value_description.h

```diff
@@ -1192,6 +1196,14 @@ struct EnumDescription: public ValueDescriptionT<Enum> {
         // TODO: description
     }
 
+    virtual const std::vector<std::string> getEnumKeys() const {
+        std::vector<std::string> res;
+        for (auto it = print.begin(); it != print.end(); ++ it) {
```

Comment:
En fait, ici, tu peux directement faire:
 
for (const auto & it: print) {
  res.push_back(it.second);
}

"it" n'est plus un iterateur a ce moment-la mais directement une paire de ton map "print".